src/components/FlowRunsAccordion.vue

```diff
@@ -35,6 +35,25 @@
   const flowIds = computed(() => flows.value?.map(flow => flow.id))
   const flowsLookup = computed(() => toMap(flows.value ?? [], 'id'))
 
+  const selectedAccordionItem: Ref<string | null> = ref(null)
+
+  watch(flowIds, () => {
+    selectedAccordionItem.value = flowIds.value?.[0] ?? null
+  })
+
+  const accordionModel = computed({
+    get() {
+      if (selectedAccordionItem.value) {
+        return selectedAccordionItem.value
+      }
+
+      return null
+    },
+    set(selected) {
+      selectedAccordionItem.value = selected
+    },
```

Comment:
it doesn't seem like this computed is needed? I think you could v-model `selectedAccordionItem` and everything would work the same

Reply:
Ahh. Indeed. haha, updating